Training/Training/Change.c

```diff
@@ -0,0 +1,40 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// Change.c
+// Program on main branch.
+// ------------------------------------------------------------------------------------------------
+#include <stdio.h>
+
+void ChangeRequired (int* balance, int* change10, int* change5, int* change2, int* change1) {
```

Comment:
Instead of the parameters change10, change5 etc., you can have a single parameter array for change.